GW2EIBuilders/Resources/combatReplayTemplates/tmplCombatReplayTargetStats.html

```diff
@@ -1,15 +1,16 @@
 ﻿<template>
     <div>
-        <combat-replay-actor-buffs-stats-component v-if="buffstoshow.length > 0" :time="time" :actorindex="targetindex" :enemy="true" :buffstoshow="buffstoshow"></combat-replay-actor-buffs-stats-component>
-        <div class="d-flex mb-1 mt-1 align-items-center">
-            <combat-replay-target-status-component :time="time" :targetindex="targetindex"></combat-replay-target-status-component>
+        <combat-replay-actor-buffs-stats-component v-if="buffstoshow.length > 0" :time="time" :actorindex="targetindex" :enemy="true" :buffstoshow="buffstoshow"/>
+        <div class="d-flex mb-1 mt-1">
+            <combat-replay-target-status-component :time="time" :targetindex="targetindex"/>
+            <combat-replay-actor-rotation-component v-if="rotation" :time="time" :actorindex="targetindex" :enemy="true"/>
         </div>
     </div>
 </template>
 
 <script>
     Vue.component("combat-replay-target-stats-component", {
-        props: ["targetindex", "time", "buffstoshow"],
+        props: ["targetindex", "time", "rotation", "buffstoshow"],
```

Comment:
I think we can make do without that option for targets, the Show Buffs/Show Rotation exists for players as with many players the UI can get quite rowdy but I don't think that is a concern for targets.